isaric/parsers/isaric-tier1.toml

```diff
@@ -198,6 +198,22 @@
     ref = "Y/N"
     can_skip = true
 
+  [subject.vaccinated_covid19]
+    combinedType = "firstNonNull"
+    fields = [
+      { fieldPattern = 'flw\d*_vacc_cov19_cmyn', values = { 1 = true, 0 = false } },
```

Comment:
I think the variable name is just flw_vacc_cov19_cmyn, not sure the field pattern is needed? Same for the vaccination dates. The combined type can be made list to capture records of all dates (and therefore we assume doses)

Reply:
keep combined type as 'set' as discussed